src/PKSim.Core/Snapshots/Mappers/OutputIntervalMapper.cs

```diff
@@ -24,6 +24,12 @@ public override Task<SnapshotOutputInterval> MapToSnapshot(ModelOutputInterval o
          });
       }
 
+      protected override bool ShouldExportParameterToSnapshot(IParameter parameter)
+      {
+         //we want to ensure that start time and end time are always exported
+         return parameter.NameIsOneOf(Constants.Parameters.START_TIME, Constants.Parameters.END_TIME) || base.ShouldExportParameterToSnapshot(parameter);
```

Comment:
Start time and end time should always be exported because their state is set to IsDefault = true when created